packages/squiggle-lang/src/rescript/Utility/E/E_Int.res

```diff
@@ -1,2 +0,0 @@
-let max = (i1: int, i2: int) => i1 > i2 ? i1 : i2
-let random = (~min, ~max) => Js.Math.random_int(min, max)
```

Comment:
I know it was used only once, but since we'll eventually have to do #248 and #227, I think we should keep this (with the intent of eventually abstracting all randomness and never use Js.Math directly).

Reply:
Makes sense. I'd rather make a single `E_Random` or similar instead of using `E_Int`.